builder/azure/chroot/shared_image_gallery_destination.hcl2spec.go

```diff
@@ -1,70 +0,0 @@
-// Code generated by "packer-sdc mapstructure-to-hcl2"; DO NOT EDIT.
```

Comment:
Was this file deleted intentionally? It's what is causing tests to fail.

Reply:
I think I meant to remove it from the pull request. But deleted it entirely. Will try to revert.

Reply:
I've reverted the delete.